years.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
@mkordas That's not clear from task definition - please read again and try to be objective (exclude any context, any obvious non-written rules  and your knowledge about copyright). 
BTW [teamed/qulice#583](https://github.com/teamed/qulice/issues/583) decision doesn't contains copyright and it was closed by you. 
I think, in this case the best way to report a new issue with explicit policy what files should be covered with copyright, what not. It is not about `.sh` - it is easy to do. But in this case we will left without copyright policy